src/constants/api.ts

```diff
@@ -0,0 +1,14 @@
+enum TRANSACTION_APIS {
```

Comment:
I'm thinking this is complicating things. Why move that to the constants? We could just have that in the explorers, that way we don't have to handle this info in various places. Furthermore, it seems this info is only used in explorers (and related tests)

Reply:
It was always in the constants, it's showing as a new file because I renamed it to TS. The only change I made is the enum. But I agree, I want to move this to explorers eventually. But I think it's another PR.